substrate/frame/contracts/src/lib.rs

```diff
@@ -101,6 +101,8 @@ mod wasm;
 pub mod chain_extension;
 pub mod debug;
 pub mod migration;
+#[cfg(feature = "std")]
```

Comment:
Is this correct? Or should it be `#[cfg(test)]`? I assume it is correct because you want to use it from non test code in the mock network? Might need a comment here why std.

Reply:
Yeah it's only used in test so with std env. 
#[cfg(test)] would make things more explicit, let me see if I can change that

Reply:
Yeah does not work, since we want to share the module, it can't be gated behind the cfg(test)

Reply:
Sorry for being pedantic here. I am sure I would scratch my head later otherwise.

```suggestion
// Can't use #[cfg(test] because it is used by the non test but std mock-network crate.
#[cfg(feature = "std")]
```

Reply:
Can you move the cfg into the module file? Then it is next to the doc comment and no one will scratch their head later when seeing this std.